todo

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+source $HOME/.todorc
+
+
+add_task(){
+
+ temp=$(cat $TODO_PATH/id.txt)
+ ID="${temp:=1}"
+ ID=$(($ID+1))
+ echo $ID >$TODO_PATH/id.txt
+ echo $ID $1 >> $TODO_PATH/todo.txt 
+ echo "$1 added sucessfuly";
+}
+
+list_task(){
+    	case $1 in
+ 	# Handle the case when the user adds both of the option
+	-a|--all)echo $(cat $TODO_PATH/todo.txt) ;;
+        -d|--done)echo $(cat $TODO_PATH/done.txt);;
+        "")echo $(cat $TODO_PATH/todo.txt)
+        echo $(cat $TODO_PATH/done.txt);;
+	*) echo bad insert ;      
+esac
+}
+
+done_tasks(){
+	id_list=$(echo $1 | tr ',' ' ')
+	for word in $id_lis
+	do
+												                        grep "^\<$word\>" $TODO_PATH/todo.txt | sponge $TODO_PATH/done.txt
+      sed "^/\<$word\>/d" $TODO_PATH/todo.txt | sponge $TODO_PATH/todo.txt					
```

Comment:
I see that you're using `$TODO_PATH` without falling back to a default value if it's not defined in `todorc` file